tests/rancher/docker_rancher.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+# Maintainer: George Gkioulis <ggkioulis@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use containers::common;
+use containers::utils;
+use rancher::utils;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my ($running_version, $sp, $host_distri) = get_os_release;
+    my $runtime = "docker";
+    install_docker_when_needed($host_distri);
+
+    setup_rancher_container(runtime => $runtime);
```

Comment:
`$runtime` variable is used once, you can remove that variable and just do 
```suggestion
    setup_rancher_container(runtime => 'docker');
```

Reply:
good catch : D